### ISSUE TITLE
WARNING: Makie.MakieLayout is deprecated

### ISSUE THREAD

**Issue by prbzrg** [MakieOrg/Makie.jl#3072]
I got these warnings in the precompilation:

```julia
┌ GLMakie [e9467ef8-e4e7-5192-8a1a-b1aee30e663a]
│  WARNING: Makie.MakieLayout is deprecatedThe module `MakieLayout` has been removed and integrated into Makie, so simply replace all usage of `MakieLayout` with `Makie`.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.MakieLayout into GLMakie.
│  WARNING: Makie._alternative_fonts is deprecated, use ALTERNATIVE_FONTS instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._alternative_fonts into GLMakie.
│  WARNING: Makie._current_figure is deprecated, use CURRENT_FIGURE instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._current_figure into GLMakie.
│  WARNING: Makie._default_font is deprecated, use DEFAULT_FONT instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._default_font into GLMakie.
│  WARNING: Makie.default_palettes is deprecated, use DEFAULT_PALETTES instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.default_palettes into GLMakie.
│  WARNING: Makie.minimal_default is deprecated, use MAKIE_DEFAULT_THEME instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.minimal_default into GLMakie.
└
```
```julia
┌ CairoMakie [13f3f980-e62b-5c42-98c6-ff1f3baf88f0]
│  WARNING: Makie.MakieLayout is deprecatedThe module `MakieLayout` has been removed and integrated into Makie, so simply replace all usage of `MakieLayout` with `Makie`.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.MakieLayout into CairoMakie.
│  WARNING: Makie._alternative_fonts is deprecated, use ALTERNATIVE_FONTS instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._alternative_fonts into CairoMakie.
│  WARNING: Makie._current_figure is deprecated, use CURRENT_FIGURE instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._current_figure into CairoMakie.
│  WARNING: Makie._default_font is deprecated, use DEFAULT_FONT instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._default_font into CairoMakie.
│  WARNING: Makie.default_palettes is deprecated, use DEFAULT_PALETTES instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.default_palettes into CairoMakie.
│  WARNING: Makie.minimal_default is deprecated, use MAKIE_DEFAULT_THEME instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.minimal_default into CairoMakie.
└
```
```julia
┌ WGLMakie [276b4fcb-3e11-5398-bf8b-a0c2d153d008]
│  WARNING: Makie.MakieLayout is deprecatedThe module `MakieLayout` has been removed and integrated into Makie, so simply replace all usage of `MakieLayout` with `Makie`.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.MakieLayout into WGLMakie.
│  WARNING: Makie._alternative_fonts is deprecated, use ALTERNATIVE_FONTS instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._alternative_fonts into WGLMakie.
│  WARNING: Makie._current_figure is deprecated, use CURRENT_FIGURE instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._current_figure into WGLMakie.
│  WARNING: Makie._default_font is deprecated, use DEFAULT_FONT instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._default_font into WGLMakie.
│  WARNING: Makie.default_palettes is deprecated, use DEFAULT_PALETTES instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.default_palettes into WGLMakie.
│  WARNING: Makie.minimal_default is deprecated, use MAKIE_DEFAULT_THEME instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.minimal_default into WGLMakie.
│  ┌ Warning: `um(tag, args...; kw...)` is deprecated, use `m(tag, args...; kw...)` instead.
│  │   caller = three_display(session::JSServe.Session{JSServe.NoConnection}, scene::Makie.Scene; screen_config::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}}) at three_plot.jl:48
│  └ @ WGLMakie C:\Users\Hossein Pourbozorg\.julia\packages\WGLMakie\3HpV9\src\three_plot.jl:48
└
```
```julia
┌ RPRMakie [22d9f318-5e34-4b44-b769-6e3734a732a6]
│  WARNING: Makie.MakieLayout is deprecatedThe module `MakieLayout` has been removed and integrated into Makie, so simply replace all usage of `MakieLayout` with `Makie`.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.MakieLayout into RPRMakie.
│  WARNING: Makie._alternative_fonts is deprecated, use ALTERNATIVE_FONTS instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._alternative_fonts into RPRMakie.
│  WARNING: Makie._current_figure is deprecated, use CURRENT_FIGURE instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._current_figure into RPRMakie.
│  WARNING: Makie._default_font is deprecated, use DEFAULT_FONT instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._default_font into RPRMakie.
│  WARNING: Makie.default_palettes is deprecated, use DEFAULT_PALETTES instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.default_palettes into RPRMakie.
│  WARNING: Makie.minimal_default is deprecated, use MAKIE_DEFAULT_THEME instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.minimal_default into RPRMakie.
└
```

**Comment by prbzrg** [comment]
Also

```julia
┌ CairoMakie [13f3f980-e62b-5c42-98c6-ff1f3baf88f0]
│  WARNING: Makie.MakieLayout is deprecatedThe module `MakieLayout` has been removed and integrated into Makie, so simply replace all usage of `MakieLayout` with `Makie`.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.MakieLayout into CairoMakie.
│  WARNING: Makie._alternative_fonts is deprecated, use ALTERNATIVE_FONTS instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._alternative_fonts into CairoMakie.
│  WARNING: Makie._current_figure is deprecated, use CURRENT_FIGURE instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._current_figure into CairoMakie.
│  WARNING: Makie._default_font is deprecated, use DEFAULT_FONT instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._default_font into CairoMakie.
│  WARNING: Makie.default_palettes is deprecated, use DEFAULT_PALETTES instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.default_palettes into CairoMakie.
│  WARNING: Makie.minimal_default is deprecated, use MAKIE_DEFAULT_THEME instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.minimal_default into CairoMakie.
└
```

**Comment by prbzrg** [comment]
And

```julia
┌ WGLMakie [276b4fcb-3e11-5398-bf8b-a0c2d153d008]
│  WARNING: Makie.MakieLayout is deprecatedThe module `MakieLayout` has been removed and integrated into Makie, so simply replace all usage of `MakieLayout` with `Makie`.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.MakieLayout into WGLMakie.
│  WARNING: Makie._alternative_fonts is deprecated, use ALTERNATIVE_FONTS instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._alternative_fonts into WGLMakie.
│  WARNING: Makie._current_figure is deprecated, use CURRENT_FIGURE instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._current_figure into WGLMakie.
│  WARNING: Makie._default_font is deprecated, use DEFAULT_FONT instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie._default_font into WGLMakie.
│  WARNING: Makie.default_palettes is deprecated, use DEFAULT_PALETTES instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.default_palettes into WGLMakie.
│  WARNING: Makie.minimal_default is deprecated, use MAKIE_DEFAULT_THEME instead.
│    likely near none:1
│  WARNING: importing deprecated binding Makie.minimal_default into WGLMakie.
│  ┌ Warning: `um(tag, args...; kw...)` is deprecated, use `m(tag, args...; kw...)` instead.
│  │   caller = three_display(session::JSServe.Session{JSServe.NoConnection}, scene::Makie.Scene; screen_config::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}}) at three_plot.jl:48
│  └ @ WGLMakie C:\Users\Hossein Pourbozorg\.julia\packages\WGLMakie\3HpV9\src\three_plot.jl:48
└
```

**Comment by SimonDanisch** [comment]
This should only come up with you explicitly enabling deprecation warnings.
Its currently unavoidable, since we deprecated those and the backend re-export Makie's functions, so they need to import and re-export those bindings, to correctly deprecate them.
We could stop exporting them in https://github.com/MakieOrg/Makie.jl/pull/2831#issuecomment-1639830792

**Comment by andreasnoack** [comment]
> This should only come up with you explicitly enabling deprecation warnings.

But that is pretty common to do on CI because most packages would like to avoid using deprecated functionality. Since this can't be controlled per module, all packages that depend on Makie are currently unable to use the useful CI functionality.

**Comment by SimonDanisch** [comment]
Closed in #3113
